openvdb_houdini/houdini/SOP_OpenVDB_Filter_Level_Set.cc

```diff
@@ -635,7 +635,7 @@ and usage examples.\n");
 
         } else if (OP_TYPE_RESHAPE == op) {
 
-            hvdb::OpenVDBOpFactory("OpenVDB Offset Level Set",
+            hvdb::OpenVDBOpFactory("VDB Offset Level Set",
```

Comment:
How about this one?  VDB Offset Level Set corresponds to VDB Reshape SDF, except that the latter lacks the new Trim option.  Should this be renamed now, or only if and when the native SOP gets the new option?

Reply:
I think we rename now. It'll always be the case that we add new features to the open-source SOPs first, then they propagate to the native SOPs in time. As long as the open-source SOPs contain at minimum all of the functionality that the native SOPs do and that the UI, inputs and parameters are not different enough between the two that it confuses people then I think this is perfectly acceptable. 

I realize there's actually a fair few more of these names that can change, I just put a list up on the JIRA - OVDB-62.